test/cli/utils_test.go

```diff
@@ -51,8 +54,20 @@ func setupPKI(t *testing.T, pw string) func() {
 
 func getFixtureImage(t testing.TB, fixtureImageName string) string {
 	t.Logf("obtaining fixture image for %s", fixtureImageName)
-	imagetest.GetFixtureImage(t, "docker-archive", fixtureImageName)
-	return imagetest.GetFixtureImageTarPath(t, fixtureImageName)
+	request := imagetest.PrepareFixtureImage(t, "docker-archive", fixtureImageName)
```

Comment:
this code was copied from stereoscope so I can add a timeout, and prove the concept locally. It makes sense to change in stereoscope